scripts/get_untriaged.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import csv
+import json
+import os
+import sys
+from datetime import datetime
+from datetime import date
+from datetime import timedelta
+
+
+# Inject project path
+sys.path.append('../')
+
+
+def fetch_untriaged(args):
+    from bugbug import bugzilla
+    today = date.today() - timedelta(days=2)
+    three_months_ago = today - timedelta(days=args.days_back)
+
+    # Set bugzilla token and download bugs
+    bugzilla.set_token(args.token)
+    bugzilla.download_bugs_between(three_months_ago, today)
+
+    # Get untriaged bugs
+    bugs = bugzilla.get_bugs()
+    untriaged_bugs = []
+    for bug in bugs:
+        for history in bug['history']:
+            for change in history['changes']:
+                if change['field_name'] == 'component' and change['removed'] == 'Untriaged':
+                    untriaged_bugs.append(bug)
+
+    with open('bugs-{}.json'.format(datetime.now().strftime('%s')), 'w') as f:
+        json.dump(untriaged_bugs, f)
+
+    return untriaged_bugs
+
+
+def run_untriaged(untriaged_bugs):
+    from bugbug.models.component import ComponentModel
+    from bugbug.models.component_nn import ComponentNNModel
+
+    models = [
+        (ComponentModel, '../componentmodel'),
+        (ComponentNNModel, '../componentnnmodel')
+    ]
+    for (model_class, model_file_name) in models:
+        rows = []
+        model = model_class.load(model_file_name)
+        for bug in untriaged_bugs:
+            p = model.classify(bug, probabilities=True)
+            url = f'https://bugzilla.mozilla.org/show_bug.cgi?id={bug["id"]}'
+
+            classifiable = p[p >= 0.7].size >= 1
+            classification = ''
+            full_component = f'{bug["product"]}::{bug["component"]}'
+
+            if classifiable:
+                print("Classifying bug with ID: {}".format(bug['id']))
+                classification = model.classify(bug)[0]
+                print("Classified bug as: {}".format(classification))
+
+            else:
+                print("Skipping bug with ID: {}".format(bug['id']))
+
+            correct_prediction = full_component == classification
```

Comment:
We should make sure we apply the same rules from https://github.com/mozilla/bugbug/blob/c0373fb6dd1f4d583b865407c8a2389787fdcfce/bugbug/models/component.py#L60, e.g. a `Core::Audio/Video: Playback` issue classified as `Core::Audio/Video` should be OK.

We can probably just extract (part of) the logic from the for loop at https://github.com/mozilla/bugbug/blob/c0373fb6dd1f4d583b865407c8a2389787fdcfce/bugbug/models/component.py#L118 into a function and use it both there and here.